### DISCUSSION TITLE
Incremental Static Regeneration for Static Path

### DISCUSSION THREAD

**Discussion by ilomon10** [vercel/next.js#47288] [Help]
### Summary

I need to update the static path, for example my current path is:
```bash
# blog/[slug].js
blog
|_ slug 1
|_ slug 2
|_ slug 3
```
and i create new blog post with slug 4
```bash
# blog/[slug].js
blog
|_ slug 1
|_ slug 2
|_ slug 3
|_ slug 4
```
for now i just call rebuild entire project with git deploy hooks

i just tried ISR but it just regenerate on generated path.

**Comment by guitheengineer** [comment]
Try using ISR with fallback: 'blocking'. It will server render on demand if the page doesn't exists.
https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration
